src/nvim/edit.c

```diff
@@ -2358,13 +2358,13 @@ void set_completion(colnr_T startcol, list_T *list)
   int save_w_wrow = curwin->w_wrow;
 
   compl_curr_match = compl_first_match;
-  if (compl_no_insert) {
+  if (compl_no_insert || compl_no_select) {
     ins_complete(K_DOWN, false);
-  } else {
-    ins_complete(Ctrl_N, false);
     if (compl_no_select) {
-      ins_complete(Ctrl_P, false);
+      ins_complete(K_UP, false);
```

Comment:
If `compl_no_select`, it does:

    ins_complete(K_DOWN, false);
    ins_complete(K_UP, false);

What is the purpose of this?